tests/unittests/OnnxImporterTest.cpp

```diff
@@ -964,11 +964,11 @@ TEST_F(OnnxImporterTest, hardsigmoid) {
   }
 
   auto *save = getSaveNodeFromDest(output);
-  HardSigmoidNode *LR =
-      llvm::dyn_cast<HardSigmoidNode>(save->getInput().getNode());
+  ClipNode *LR =
+      llvm::dyn_cast<ClipNode>(save->getInput().getNode());
   ASSERT_TRUE(LR);
-  EXPECT_FLOAT_EQ(LR->getAlpha(), 0.1666666667);
-  EXPECT_FLOAT_EQ(LR->getBeta(), 0.500000001);
+  EXPECT_FLOAT_EQ(LR->getMax(), 1.0);
```

Comment:
This part is also changed, since the node type is `ClipNode`. We no longer tell what `alpha` and `beta` are.

Reply:
One way we could do this is walk the graph to check the values of `alpha` and `beta`. E.g. 
```
  AddNode *addBeta = llvm::dyn_cast<AddNode>(LR->getInput());
  ASSERT_TRUE(addBeta);
  SplatNode *betaSplat = llvm::dyn_cast<SplatNode>(addBeta->getRHS());
  ASSERT_TRUE(betaSplat);
  EXPECT_EQ(betaSplat->getValue(), 0.500000001);
  // (do the same for alpha via the LHS input of addBeta)
```

Reply:
done
c5bd996349b8666a15b03555008630172166dbe4